Bio/motifs/jaspar/__init__.py

```diff
@@ -313,6 +315,11 @@ def _read_jaspar(handle):
 
 
 def calculate_pseudocounts(motif):
+    """Calculate pseudocounts, for compatibility with Perl TFBS modules.
```

Comment:
I think we can drop "for compatibility with Perl TFBS modules". This is a commonly used method to calculate pseudocounts; it's not specific to the Perl TFBS modules.

Reply:
Ok, is removed now, Thank You.